core/services/chainlink/config_telemetry.go

```diff
@@ -41,3 +43,17 @@ func (b *telemetryConfig) TraceSampleRatio() float64 {
 	}
 	return *b.s.TraceSampleRatio
 }
+
+func (b *telemetryConfig) EmitterBatchProcessor() bool {
+	if b.s.EmitterBatchProcessor == nil {
+		return false
```

Comment:
should we default this to `true`? we'd always want it on in prod

Reply:
yep, Im trying to figure out if defaults should be set here or in some other place

Reply:
The trick https://github.com/smartcontractkit/chainlink/pull/15173/commits/b0d9f6d1d207cdb9e5e7e2420197278a9a3c4abb

Reply:
It can be OK to have defaults like this for specific cases, but the general pattern is to have a default value specified in the TOML docs which will be set and validated non-nil by tests.